### ISSUE TITLE
FormBuilderSlider does not show up

### ISSUE THREAD

**Issue by alice-0-kim** [danvick/flutter_form_builder#80]
I started with the example snippet of code from README, and it does not seem to be working. So, I am wondering if I'm missing some dependencies/attributes in order to use it correctly.
Here is the code I have in my project:
```dart
final GlobalKey<FormBuilderState> _fbKey = GlobalKey<FormBuilderState>();
...
FormBuilder(
  key: _fbKey,
  autovalidate: true,
  child: Column(
    children: <Widget>[
      ...
      FormBuilderSlider(
        attribute: "slider",
        min: 0.0,
        max: 10.0,
        initialValue: 7.0,
        divisions: 20,
        decoration: InputDecoration(
          labelText: "Number of somethings",
          fillColor: themeColor,
        ),
      ),
      ...
    ],
  ),
)
```

Currently, the code is compiling and I do not get any errors in the log. However, I do not see the slider part of the form. It is only invisible; I can still drag the 'invisible' slider around to change the value:
<img width="184" alt="20190618164629" src="https://user-images.githubusercontent.com/28884850/59727401-638bec80-91ea-11e9-9715-370ca939a079.png">

Dependencies in `pubspec.yaml`:
```yaml
dependencies:
  flutter:
    sdk: flutter
  charts_flutter: 0.6.0
  flutter_xlider: 2.4.3
  flutter_email_sender: 2.0.3
  url_launcher: ^5.0.3
  flutter_form_builder: 3.2.0
  cupertino_icons: ^0.1.2
```

Let me know if you need any other information or clarification. Thanks in advance!

**Comment by danvick** [comment]
Hi Alice,
Sorry for the late response.
Is there any chance that your primary color is `white`?
The color of the slider usually takes the app's theme `primaryColor`

**Comment by danvick** [comment]
Thanks to this issue that you have raised, I have noticed some missing bits in the package which have been fixed as of `v3.2.1`. Feel free to try it out.
You can now change your `activeColor` and `inactiveColor` of the `Slider` widget.

Your contribution is appreciated, Alice.

**Comment by alice-0-kim** [comment]
You're right. The `primaryColor` is set to `Colors.white`. Thank you so much for your time, much appreciated!
